utils/flare.py

```diff
@@ -328,50 +355,56 @@ def _can_read(cls, f, output=True, warn=True):
                 log.warn("  * not readable - {0}".format(f))
             return False
 
-    # Return path to a temp file without comment
-    def _strip_comment(self, file_path):
-        fh, temp_path = tempfile.mkstemp(prefix='dd')
-        atexit.register(os.remove, temp_path)
-        with os.fdopen(fh, 'w') as temp_file:
-            with open(file_path, 'r') as orig_file:
-                for line in orig_file.readlines():
-                    if not self.COMMENT_REGEX.match(line):
-                        temp_file.write(re.sub(self.APIKEY_REGEX, self.REPLACE_APIKEY, line))
-
-        return temp_path
-
-    # Remove password before collecting the file
-    def _add_clean_confd(self, file_path):
+    def _add_clean_conf(self, file_path, target_dir, credential_patterns=None):
```

Comment:
If you use `credential_patterns=[]` in the `_strip_credentials()` function def, you should here too for consistency.